daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExceptionTest.scala

```diff
@@ -533,21 +539,70 @@ class ExceptionTest extends AnyWordSpec with Inside with Matchers with TableDriv
       val transactionSeed: crypto.Hash = crypto.Hash.hashPrivateKey("transactionSeed")
 
       val testCases = Table[String, Boolean](
-        ("update", "caught"),
-        ("exercise @M:T BodyCrash cid ()", true),
-        ("exercise @M:T ControllersCrash cid ()", false),
-        ("exercise @M:T ObserversCrash cid ()", false),
-        ("exercise_interface @M:I BodyCrash (COERCE_CONTRACT_ID @M:T @M:I cid) ()", true),
-        ("exercise_interface @M:I ControllersCrash (COERCE_CONTRACT_ID @M:T @M:I cid) ()", false),
-        ("exercise_interface @M:I ObserversCrash (COERCE_CONTRACT_ID @M:T @M:I cid) ()", false),
+        ("udpate", "caught"),
+        """
+            ubind 
+              cid : ContractId M:T <- create @M:T (M:T {party = sig, viewFails = False}) 
+            in exercise @M:T BodyCrash cid ()
+        """ -> true,
+        """
+            ubind 
+              cid : ContractId M:T <- create @M:T (M:T {party = sig, viewFails = False})
+            in exercise @M:T ControllersCrash cid ()
+        """ -> false,
```

Comment:
I found these tests a bit hard to read, they are all very similar and you have to spot the differences. I think a "title" column would help, e.g.

```suggestion
        ( "Cannot catch exception thrown in choice controllers",
          """
              ubind 
                cid : ContractId M:T <- create @M:T (M:T {party = sig, viewFails = False})
              in exercise @M:T ControllersCrash cid ()
          """,
          false,
        )
```